src/dunst.c

```diff
@@ -72,12 +72,12 @@ void check_timeouts(void)
                         continue;
                 }
 
+                /* shift iter before we possibly delete the current notification */
+                iter = iter->next ? iter->next : NULL;
```

Comment:
Small nitpick, but why use a conditional here? If `iter->next` is `NULL`, `iter` will be assigned to `NULL` anyway.